CMakeLists.txt

```diff
@@ -285,7 +285,7 @@ endif()
 
 if(Qt5Core_FOUND)
   set_package_properties(Qt5Core PROPERTIES TYPE REQUIRED)
-  find_package(Qt5 NO_MODULE REQUIRED COMPONENTS Gui Network)
+  find_package(Qt5 NO_MODULE REQUIRED COMPONENTS Gui Network OpenGL)
```

Comment:
Correct, I was mistaken by thinking headers like <QOpenGLContext> require QtOpenGL where in fact they come from Gui.